include/swift/AST/DiagnosticsParse.def

```diff
@@ -2021,6 +2015,9 @@ ERROR(macro_expansion_decl_expected_macro_identifier,none,
 
 ERROR(parser_round_trip_error,none,
       "source file did not round-trip through the Swift Swift parser", ())
+ERROR(parser_new_parser_errors,none,
+      "Swift Swift parser generated errors for code that C++ parser accepted",
```

Comment:
Similar to my comment on another PR, I think `Swift Swift parser` is confusing. What about `SwiftParser` or `SwiftParser implemented in Swift`?